bootstrap/src/main/java/com/navercorp/pinpoint/bootstrap/config/ProfilerConfig.java

```diff
@@ -140,6 +140,8 @@ public static ProfilerConfig load(String pinpointConfigFileName) throws IOExcept
     private boolean memcached = true;
     private boolean memcachedKeyTrace = false;
     
+    private Filter<String> jettyExcludeUrlFilter = new SkipFilter<String>();
```

Comment:
If I'm not mistaken I needed this field as a default for filtering within the ServerHandleInterceptor:
```
        final String requestURI = request.getRequestURI();
        if (excludeUrlFilter.filter(requestURI)) {
            if (isTrace) {
                logger.trace("filter requestURI:{}", requestURI);
            }
            return null;
        }
```